rest-client-reactive-quickstart/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+country-api/mp-rest/url=https://restcountries.eu/rest
+country-api/mp-rest/scope=javax.inject.Singleton
```

Comment:
At some point, we should make that the default no?
I always have this line.

@mkouba @geoand @Ladicek

Reply:
I agree, I never understood why it's not... The spec maybe?

Reply:
Actually, it's best if it's `@ApplicationScoped` in order to make mocking possible

Reply:
Per the specification https://download.eclipse.org/microprofile/microprofile-rest-client-2.0/microprofile-rest-client-spec-2.0.html#restcdi, the default scope is `@Dependent`.

Reply:
I'll drop it, we have `@ApplicationScoped` by default which makes sense.

Reply:
done

Reply:
@Ladicek rest-client-reactive uses `@ApplicationScoped` by default, see https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/rest-client-reactive.adoc#known-limitations

Reply:
Didn't know that, thanks for info.